### PR TITLE
Lotame add vendor

### PR DESCRIPTION
**Pull request by Tonsil**
References: https://github.com/ampproject/amphtml/issues/3020

### CODE REVIEW

File: examples/analytics.amp.html [hunk 1]
```diff
@@ -218,6 +218,25 @@
...
+  Please replace the value for "account" (12345)
+  with the correct Lotame Client ID.
+-->
+<amp-analytics type="lotame">
+<script type="application/json">
+{
+  "vars": {
+    "account": 12345
+  },
+  "extraUrlParams": {
```
Comment (avimehta) [inline examples/analytics.amp.html]: adding extraUrlParams is not needed here. That is a platform level thing and publishers are probably aware of them. Plus, you could add more meaningful examples to lotame product documentation, i any.

File: extensions/amp-analytics/0.1/vendors.js [hunk 2]
```diff
@@ -328,6 +328,23 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'lotame': {
+    'requests': {
+      'pageview': 'https://bcp.crwdcntrl.net/amp?c=${account}&pv=y'
```
Comment (avimehta) [inline extensions/amp-analytics/0.1/vendors.js]: trailing commas here and below.

**Comment by avimehta** [comment]
please fix the lint errors, test that is failing.  rest looks good to me.

File: examples/analytics.amp.html (hunk 1 above)
Comment (Tonsil) [inline examples/analytics.amp.html]: The extraUrlParams were added as an example for our clients. They were left intentionally generic to illustrate that they could be anything and that the specifics are dependent on case. Would you prefer I remove them?

**Comment by Tonsil** [comment]
@avimehta - I made the changes for the trailing commas. Did I need to do anything else?

File: examples/analytics.amp.html (hunk 1 above)
Comment (avimehta) [inline examples/analytics.amp.html]: i think it is fine.

Comment (avimehta) [inline examples/analytics.amp.html]: I think keeping it here is fine. But can you change the values to something meaningful? What do you expect publishers to use this for most?

**Comment by avimehta** [comment]
yes, you'll need to make a change to vendors-requests.json to make the test pass. Once you do that and tests pass, please squash your commits and rebase.

**Comment by Tonsil** [comment]
Ok, fixed the test, squashed the commits, it should be good to go.

File: extensions/amp-analytics/0.1/test/vendor-requests.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
   },
   "colanalytics": {
     "host": "https://ase.clmbtech.com",
-    "base": "https://ase.clmbtech.com/message",
+    "base": "https://ase.clmbtech.com/message",
```
Comment (avimehta) [inline extensions/amp-analytics/0.1/test/vendor-requests.json]: i think an extra space got added here.

**Comment by Tonsil** [comment]
How about now?

**Comment by avimehta** [comment]
@dvoytenko Can you start the tests again? I think it is unrelated to the change in the PR.

**Comment by dvoytenko** [comment]
@avimehta started.

**Comment by dvoytenko** [comment]
@avimehta all good. feel free to merge.

**Pull request merged**